botocore/model.py

```diff
@@ -1,6 +1,7 @@
 """Abstractions to interact with service models."""
 from collections import defaultdict
 
+from botocore import xform_name
```

Comment:
Nah. It did not get cleaned up after I removed the ``client_name_to_operation_name`` function I added to the model. Will remove